eoverdracht/leveranciersspecificatie.md

```diff
@@ -340,23 +340,79 @@ Zie [RFC003](https://nuts-foundation.gitbook.io/drafts/rfc/rfc003-oauth2-authori
 
 Omdat de identiteit van de gebruiker en de opvragende partij zijn vastgelegd in het access token kunnen deze ook worden gelogd door het bronsysteem, waarmee voldaan is aan de NEN7513 eisen inzake logging. Deze logging kan dan ook \(bijvoorbeeld via een PGO\) inzichtelijk gemaakt worden voor de patiënt.
 
-#### 4.2.7 FHIR Consent inhoud
+#### 4.2.7 Service autorisatie
 
-In het consent record moet in ieder geval het volgende zijn opgenomen:
+In het autorisatie record moet in ieder geval het volgende zijn opgenomen:
 
-* Een identifier van het consent record
+* Een identifier van het autorisatie record
 * De bronhouder
 * De ontvangende partij
 * Een verwijzing naar het aanmeldbericht zodat deze opgehaald kan worden op de aangegeven locatie
-* De grondslag reden \(eOverdrachtaanmeldbericht\)
+* Een verwijzing naar de bijbehorende Task. De ontvangende partij moet immers rechten hebben om de Task te mogen updaten 
+* Het type grondslag (impliciet)
```

Comment:
impliciet == verondersteld?

Reply:
Ja, dit is verwarrend

Reply:
Hier aangepast naar *veronderstelde teostemming*